proto/validator/accounts/v2/web_api.proto

```diff
@@ -341,3 +347,13 @@ message BackupAccountsResponse {
     // Zip file containing backed up keystores.
     bytes zip_file = 1;
 }
+
+message DeleteAccountsRequest {
+    // List of public keys to delete.
+    repeated bytes delete_public_keys = 1;
```

Comment:
`delete_public_keys` sounds like a good function name, but parameter names should be nouns. I would prefer simply `public_keys`.

Reply:
agreed. Although, I was trying to be consistent with the accounts.Config . Should i change the protobuf? 

```
type Config struct {
  | Wallet           *wallet.Wallet
  | Keymanager       keymanager.IKeymanager
  | DeletePublicKeys [][]byte
  | }
```

Reply:
It's OK if they are not the same. Config's name is also a bit awkward, I think `PublicKeysToDelete` makes more sense.

Reply:
> So it looks like [7348624](https://github.com/prysmaticlabs/prysm/commit/7348624d32a089fe22421df7a7c15b74229dafa9) removed an import from `proto/beacon/rpc/v1_gateway/debug.pb.gw.go` that was actually needed. Also the alias in import `v1alpha1 "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"` in `proto/validator/accounts/v2_gateway/web_api.pb.go` should be changed to `eth` (resulting in `eth "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"`).

I think its because i ran update proto scripts before goimports.  It is fixed now.